messages/models.tsp

```diff
@@ -85,16 +101,17 @@ model MessageObject {
   assistant_id: string | null;
 
   /**
-   * If applicable, the ID of the [run](/docs/api-reference/runs) associated with the authoring of
-   * this message.
+   * The ID of the [run](/docs/api-reference/runs) associated with the creation of this message.
+   * Value is `null` when messages are created manually using the create message or create thread
+   * endpoints.
    */
   run_id: string | null;
 
   /**
-    * A list of [file](/docs/api-reference/files) IDs that the assistant should use. Useful for
-    * tools like retrieval and code_interpreter that can access files. A maximum of 10 files can be
-    * attached to a message.
-    */
+   * A list of [file](/docs/api-reference/files) IDs that the assistant should use.
+   * Useful for tools like retrieval and code_interpreter that can access files.
+   */
+  @minItems(1)
```

Comment:
I wonder: Is it correct to say `@minItems(1)` when the default value is an empty array?